tools/DevInsights/DevHome.DevInsights/Controls/PISeparator.xaml.cs

```diff
@@ -4,7 +4,7 @@
 using Microsoft.UI.Xaml;
 using Microsoft.UI.Xaml.Controls;
 
-namespace DevHome.PI.Controls;
+namespace DevHome.DevInsights.Controls;
```

Comment:
Should the name of the file be DevInsightsSeperator?

Reply:
Yes